### PR TITLE
support auth delegating

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,12 +65,36 @@ make run
...
+
+1. Login to your cluster with `oc login`
+
+2. Create a new context in your cluster kubeconfig
+
+    ```bash
+    route=$(oc -n multicluster-controlplane get route multicluster-controlplane -o jsonpath='{.status.ingress[0].host}')
+
+    # if you want to add the ca of your cluster kube-apiserver, using the command:
+    # oc config set-cluster multicluster-controlplane --server="https://${route}" --embed-certs --certificate-authority=<the ca path of your cluster kube-apiserver>
```
Comment (clyang82) [inline README.md:86]: can we provide a command to get the openshift ca? if so, we can remove `--insecure-skip-tls-verify`.

Comment (skeeey) [reply README.md:86]: yes, we can, will update this part

Comment (skeeey) [reply README.md]: updated

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**